src/main/java/com/example/ludogoriesoft/lukeriaerpapi/services/InvoiceService.java

```diff
@@ -18,6 +19,25 @@ public class InvoiceService {
     private final InvoiceRepository invoiceRepository;
     private final ModelMapper modelMapper;
 
+    public Long findLastInvoiceNumberStartingWithTwo()  {
+        String prefix = "2";
+
+        List<String> lastInvoiceNumbers = invoiceRepository.findLastInvoiceNumberStartingWith(prefix);
+
+        if (lastInvoiceNumbers.isEmpty()) {
+           return 2000000000L;
```

Comment:
Do not use magic numbers:
https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad